### PR TITLE
Support retrieving OTP codes as a new subcommand

### PR DESCRIPTION
**Pull request by RobertPlant**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ passwordstore (`pass`) \
 - `new` - Generate a new password
 - `del` - Delete a password
 - `insert` - Insert a password
+- `otp` - Get OTP passcodes
```
Comment (rupansh) [inline README.md:34]: A link to https://github.com/tadfisher/pass-otp would be nice

**Comment by rupansh** [comment]
Hello! It will be nice if you could implement the other pass-otp subcommands as well!
Consider moving the `pass-otp` specific code to `src/otp.rs`

**Comment by RobertPlant** [comment]
I have addressed your feedback, let me know if you have any other suggestions

**Comment by rupansh** [comment]
Hi i improved the argument parsing a bit. I don't use pass-otp so I am not sure what's supposed to be the correct behavior. Can you give it a quick test?

**Comment by RobertPlant** [comment]
Hey, thanks for the improvement, OTP retrieval and insertion are still working with your updates

**Pull request merged**